tests/utils.go

```diff
@@ -885,6 +893,29 @@ func NewRandomVMIWithEphemeralDiskAndUserdata(containerImage string, userData st
 	return vmi
 }
 
+func NewRandomVMIWithEphemeralDiskAndUserdataAndNS(containerImage string, namespace string, userData string) *v1.VirtualMachineInstance {
+	vmi := NewRandomVMIWithEphemeralDiskAndNS(containerImage, namespace)
```

Comment:
you can use 
```
vmi := NewRandomVMIWithEphemeralDiskAndUserdata(containerImage, userData)
vmi.Namespace = namespace

retrun vmi
```

it will save some lines of code and you can to remove `NewRandomVMIWithEphemeralDiskAndNS` at all

Reply:
Updated to use the way you mentioned and remove the func. Thanks